src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -203,12 +203,12 @@ public override string Description
         {
             get
             {
-                if (DisplayMetadata.Description == null)
+                if (DisplayMetadata.Description != null)
                 {
-                    return null;
+                    return DisplayMetadata.Description();
                 }
 
-                return DisplayMetadata.Description();
+                return null;
```

Comment:
We tend to leave existing style alone -- except in the case of bizarre and hard to maintain line wrapping. Why did this and `DisplayName` change at all?